src/hooks/stores/preferences.ts

```diff
@@ -31,6 +38,14 @@ export class Preferences implements IPreferences {
   onboarding!: "new" | "complete";
   @observable
   darkMode!: "light" | "dark" | "system";
+  @observable
+  fonts?: {
```

Comment:
While each font value may not be present, the fonts object itself is not optional.